python/ray/util/sgd/v2/trainer.py

```diff
@@ -444,6 +409,66 @@ def to_tune_trainable(self, train_func: Callable[[Dict[str, Any]], T]
                                       self._num_workers, self._use_gpu,
                                       self._resources_per_worker)
 
+    def to_workers(self, train_cls: Type, *args, **kwargs) -> "Workers":
```

Comment:
nit: IMO renaming this to `start_workers` makes sense as a parallel to `start` that exists.

```suggestion
    def start_workers(self, train_cls: Type, *args, **kwargs) -> "Workers":
```

Reply:
Discussed offline- to me `to_workers` makes more sense since it implies that something new is being returned that you're supposed to work with instead of working with Trainer. `start_workers` might be confusing with `start`.